all/environments/multiagent_pettingzoo_test.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import torch
+from all.environments import MultiagentPettingZooEnv
+from pettingzoo.mpe import simple_world_comm_v2
+
+
+class MultiagentPettingZooEnvTest(unittest.TestCase):
```

Comment:
These tests could be a little more thorough. Ideally, all of the public methods are tested.